src/System.Reflection.TypeExtensions/ref/System.Reflection.TypeExtensions.cs

```diff
@@ -15,12 +15,12 @@ public static partial class AssemblyExtensions
     }
     public static partial class EventInfoExtensions
     {
-        public static System.Reflection.MethodInfo GetAddMethod(this System.Reflection.EventInfo eventInfo) { throw null; }
-        public static System.Reflection.MethodInfo GetAddMethod(this System.Reflection.EventInfo eventInfo, bool nonPublic) { throw null; }
-        public static System.Reflection.MethodInfo GetRaiseMethod(this System.Reflection.EventInfo eventInfo) { throw null; }
-        public static System.Reflection.MethodInfo GetRaiseMethod(this System.Reflection.EventInfo eventInfo, bool nonPublic) { throw null; }
-        public static System.Reflection.MethodInfo GetRemoveMethod(this System.Reflection.EventInfo eventInfo) { throw null; }
-        public static System.Reflection.MethodInfo GetRemoveMethod(this System.Reflection.EventInfo eventInfo, bool nonPublic) { throw null; }
+        public static System.Reflection.MethodInfo?  GetAddMethod(this System.Reflection.EventInfo eventInfo) { throw null; }
```

Comment:
Nit, this has an extra space after the `?` -- did you do it manually or used `GenAPI`?

Reply:
I forgot about GenAPI. Remind me how to do it? I ran `msbuild /t:GenerateReferenceAssemblySource` in both ref and src and it didn't seem to do it.

Reply:
@buyaa-n ?

Reply:
it should work from src folder, also the option is little different: 
`msbuild /t:GenerateReferenceSource`

Reply:
Yeah that should do it.